sos/report/plugins/gluster.py

```diff
@@ -104,7 +104,7 @@ def setup(self):
                                      "glusterd processes")
             else:
                 self.soslog.warning("Unable to generate statedumps, no such "
-                                    "directory: %s" % self.statedump_dir)
+                                    "directory: {self.statedump_dir}")
```

Comment:
`W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)`

So replaced with f-string here .